### PR TITLE
IdP Guides - Card Selection Page

### PR DESCRIPTION
**Pull request by johnbaldo-okta**
## Description:
- Adding a landing page in Guides for IdPs with card selection. Vision is for it to be search and filterable like Error Codes, but just plopping them on the page for now.
- Modified the card component to take an image in addition to an icon. If both are provided the icon will display and not the image.

### Future enhancements:
- Search / filter by IdP name
- External link to Smart Card IdP under Enterprise IdPs

Screen Shot 2021-10-29 at 6 29 49 PM

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md [hunk 1]
```diff
@@ -8,61 +8,41 @@ As a developer building a custom application, you want to give your users the fr
...
 
 > **Note:** Social and OpenID Connect Identity Providers store access tokens that allow subsequent calls to Identity Providers after the user is authorized. For example, the token may contain the permission to add events to a user's Google calendar. After the user is authorized, the token can be used on additional calls to add events to a user's Google calendar on the user's behalf.
 
-* You can also configure federation [between Okta orgs](/docs/guides/add-an-external-idp/oktatookta/main/) using OpenID Connect or SAML.
+-   You can also configure federation [between Okta orgs](/docs/guides/add-an-external-idp/oktatookta/main/) using OpenID Connect or SAML.
```
Comment (johnbaldo-okta) [inline packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md:25]: I think my SublimeText had some auto-formatting turned on. If it's a problem, I can go back and remove it.

File: packages/@okta/vuepress-theme-prose/assets/css/components/_card.scss [hunk 2]
```diff
@@ -103,6 +103,10 @@
     }
   }
 
+  .card--header-img {
+    width: 4rem;
```
Comment (johnbaldo-okta) [inline packages/@okta/vuepress-theme-prose/assets/css/components/_card.scss:107]: May need to adjust this to work with horizontal logos (Yahoo Japan could be a test. I don't think they have an icon).

File: packages/@okta/vuepress-site/code/dotnet/blazor/index.md [hunk 3]
```diff
@@ -38,18 +38,14 @@ New to Okta? Our Blazor WebAssembly & Server-Side samples will walk you through
 	<span class='fa fa-github'></span> <span>Blazor WebAssembly (WASM) Sample App</span>
 </a>
 
-
 ## Other .NET Libraries
 
-<Card href="" :showHeaderIcon=false cardTitle="Okta management SDK for .NET" :showFooter=false>Enable your ASP.NET application to work with Okta via OAuth 2.0/OIDC</Card>
+<Card href="" cardTitle="Okta management SDK for .NET">Enable your ASP.NET application to work with Okta via OAuth 2.0/OIDC</Card>
```
Comment (johnbaldo-okta) [inline packages/@okta/vuepress-site/code/dotnet/blazor/index.md:43]: This card doesn't go anywhere

File: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md [hunk 4]
```diff
@@ -8,61 +8,41 @@ As a developer building a custom application, you want to give your users the fr
...
-- [Salesforce](/docs/guides/social-login/salesforce/main)
-- [Yahoo](/docs/guides/social-login/yahoo/main)
-- [Yahoo Japan](/docs/guides/social-login/yahoojp/main)
-
-Additionally, Okta supports the following Identity Providers:
-
-- [Okta to Okta](/docs/guides/add-an-external-idp/oktatookta/main/)
-- [OpenID Connect](/docs/guides/add-an-external-idp/openidconnect/main/)
-- [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/)
+[View and configure Okta's Identity Provider integrations](/docs/guides/identity-providers/)
```
Comment (johnbaldo-okta) [inline packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md:11]: Any way to make this pop out a bit? Button?

File: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md:25]: either the * or the - are read as bullets...so you're all good.

File: packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md (hunk 4 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md:11]: We can do that - but I believe that @albertchen-okta's team would be the people to ask for that.

**Review by susanharper-okta — APPROVED** [review]
just one tiny change - and some comments. Approving to not hold you up.

**Review by timhowie-okta — APPROVED** [review]
lgtm

**Pull request merged**